test/util.js

```diff
@@ -33,9 +33,7 @@ function fixtureToInstace ({url}, options) {
   url = parseUrl(url)
 
   return new GitHub(merge(options, {
-    host: url.host,
-    protocol: url.protocol.replace(/:$/, ''),
-    pathPrefix: url.path
+    baseUrl: url.protocol.replace(/:$/, '') + url.host + '/' + url.path
```

Comment:
@aryannarora very close! With the new `baseUrl` parameter, we don’t need to parse `url` option any more, instead we can pass it directly as `baseUrl`. We only parsed the URL because we had to pass `host`, `protocol` and `pathPrefix` separately before

Reply:
@gr2m This whole time i was thinking that the response is in such a way that we have to parse it.
Thanks for this clarification. :)